pulp_smash/utils.py

```diff
@@ -86,6 +86,26 @@ def os_is_f27(cfg, pulp_host=None):
     return response.returncode == 0
 
 
+def fips_is_enabled(cfg, pulp_host=None):
```

Comment:
Please implement as two separate methods, e.g. `fips_is_supported` and `fips_is_enabled`. My developer spidey senses are tingling.